lib/dependencies.sh

```diff
@@ -33,13 +33,30 @@ log_build_scripts() {
   local build=$(read_json "$BUILD_DIR/package.json" ".scripts[\"build\"]")
   local heroku_prebuild=$(read_json "$BUILD_DIR/package.json" ".scripts[\"heroku-prebuild\"]")
   local heroku_postbuild=$(read_json "$BUILD_DIR/package.json" ".scripts[\"heroku-postbuild\"]")
+  local postinstall=$(read_json "$BUILD_DIR/package.json" ".scripts[\"heroku-postbuild\"]")
 
   if [ -n "$build" ]; then
     mcount "scripts.build"
 
     if [ -z "$heroku_postbuild" ]; then
       mcount "scripts.build-without-heroku-postbuild"
     fi
+
+    if [ -z "$postinstall" ]; then
+      mcount "scripts.build-without-postinstall"
+    fi
+
+    if [ -z "$postinstall" ] && [ -z "$heroku_postbuild" ]; then
+      mcount "scripts.build-without-other-hooks"
+    fi
+  fi
+
+  if [ -n "$postinstall" ]; then
+    mcount "scripts.postinstall"
+
+    if [ "$postinstall" == "npm run build" ]; then
```

Comment:
Should this and the below also check for `yarn run build` and the abbreviated form, `yarn build` ?

Reply:
I was feeling lazy. I'll add it :)

There are probably a ton of people who just have a build script copy+pasted in `postinstall` :(